campapp/main.c

```diff
@@ -39,8 +39,13 @@ void infoscreen(){
     lcdPrintln("DOWN: DFU mode");
     lcdNl();
     lcdPrintln("UP:   MSC");
-    lcdPrintln("  (to copy files");
-    lcdPrintln("     to badge)");
+    lcdPrintln("(copy files to badge)");
```

Comment:
On my device this does not fit in one line. Why has the line break been removed?
(Battery status still would fit)